api/usersd/service.go

```diff
@@ -561,6 +566,146 @@ func (s *Service) ArchiveRetrievalLogs(req *pb.ArchiveRetrievalLogsRequest, serv
 	return nil
 }
 
+func (s *Service) ListFilRewards(ctx context.Context, req *pb.ListFilRewardsRequest) (*pb.ListFilRewardsResponse, error) {
+	account, _ := mdb.AccountFromContext(ctx)
+	org := account.Org
+	user := account.User
+
+	if org == nil {
```

Comment:
Seems like all these methods should be on the hub service if we're requiring an org. All the existing `userd` methods are accessible to devs and users. Maybe you put them here for something in the future?

Reply:
No, I asked you where to put them and you told me `usersd` haha. If it's better on `hubd`, that's great and I'll move them.

Reply:
Ok will move.

Reply:
Moved to hubd service and client.